package.json

```diff
@@ -23,102 +23,102 @@
-    "nav-frontend-paneler": "^2.0.25",
-    "nav-frontend-paneler-style": "^0.3.33",
-    "nav-frontend-popover": "0.0.43",
-    "nav-frontend-popover-style": "0.0.14",
-    "nav-frontend-skjema": "^3.0.44",
-    "nav-frontend-skjema-style": "^2.0.27",
-    "nav-frontend-spinner": "^2.0.16",
+    "moment": "^2.29.4",
+    "nav-frontend-alertstriper": "^4.0.2",
+    "nav-frontend-alertstriper-style": "^3.0.2",
+    "nav-frontend-chevron": "^1.0.30",
+    "nav-frontend-chevron-style": "^1.0.4",
+    "nav-frontend-core": "^6.0.1",
+    "nav-frontend-ekspanderbartpanel": "^4.0.4",
+    "nav-frontend-ekspanderbartpanel-style": "^2.0.2",
+    "nav-frontend-etiketter": "^3.0.3",
+    "nav-frontend-etiketter-style": "^2.0.3",
+    "nav-frontend-hjelpetekst": "^3.1.3",
+    "nav-frontend-hjelpetekst-style": "^3.0.2",
+    "nav-frontend-ikoner-assets": "^3.0.1",
+    "nav-frontend-js-utils": "^1.0.20",
+    "nav-frontend-knapper": "^3.1.3",
+    "nav-frontend-knapper-style": "^2.1.2",
+    "nav-frontend-lenkepanel": "^2.0.2",
+    "nav-frontend-lenkepanel-style": "^2.0.2",
+    "nav-frontend-lenker": "^2.0.2",
+    "nav-frontend-lenker-style": "^2.0.2",
+    "nav-frontend-lesmerpanel": "2.0.3",
+    "nav-frontend-lesmerpanel-style": "2.0.2",
+    "nav-frontend-lukknapp": "^2.0.2",
+    "nav-frontend-lukknapp-style": "^2.0.2",
+    "nav-frontend-modal": "^2.0.2",
+    "nav-frontend-modal-style": "^2.0.2",
+    "nav-frontend-paneler": "^3.0.2",
+    "nav-frontend-paneler-style": "^2.0.2",
+    "nav-frontend-popover": "2.0.4",
+    "nav-frontend-popover-style": "2.0.2",
+    "nav-frontend-skjema": "^4.0.6",
+    "nav-frontend-skjema-style": "^3.0.3",
+    "nav-frontend-spinner": "^3.0.1",
     "nav-frontend-spinner-style": "^1.0.2",
-    "nav-frontend-tabs": "0.0.60",
-    "nav-frontend-tabs-style": "0.0.39",
-    "nav-frontend-typografi": "^2.0.39",
-    "nav-frontend-typografi-style": "^1.0.32",
-    "nav-frontend-veileder": "^2.0.37",
-    "nav-frontend-veileder-style": "^2.0.27",
-    "nav-frontend-veilederpanel": "^0.1.46",
-    "nav-frontend-veilederpanel-style": "0.0.43",
+    "nav-frontend-tabs": "2.0.3",
+    "nav-frontend-tabs-style": "2.0.2",
+    "nav-frontend-typografi": "^4.0.2",
+    "nav-frontend-typografi-style": "^2.0.2",
+    "nav-frontend-veileder": "^3.0.3",
+    "nav-frontend-veileder-style": "^3.0.2",
+    "nav-frontend-veilederpanel": "^2.0.3",
+    "nav-frontend-veilederpanel-style": "2.0.2",
     "node-cache": "^5.1.2",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^3.2.10",
     "npm-run-all": "^4.1.5",
-    "prop-types": "^15.7.2",
-    "react": "^17.0.1",
-    "react-collapse": "^5.1.0",
-    "react-day-picker": "^7.4.8",
-    "react-dom": "^17.0.1",
-    "react-intl": "^5.17.4",
-    "react-modal": "^3.12.1",
-    "react-moment": "^0.9.7",
-    "react-motion": "^0.5.2",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.4",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.0",
```

Comment:
Kan vi bruke nyeste patch for denne + react-dom? (17.0.2)

Reply:
Done!